### ISSUE TITLE
Support ARIA 1.3's aria-description property

### ISSUE THREAD

**Issue by aleventhal** [w3c/core-aam#60]
The precedence order is:
aria-describedby
aria-description

**Comment by aleventhal** [comment]
Pull request for aria-description in ARIA spec:
https://github.com/w3c/aria/pull/1137

**Comment by aleventhal** [comment]
@joanmarie is it time to review these CORE-AAM changes?

**Comment by joanmarie** [comment]
This is merged already in Core-AAM. @aleventhal is there anything else we need to do before closing this issue?

**Comment by aleventhal** [comment]
There is one remaining issue regarding the AX API mappings being incorrect/incomplete. I filed https://github.com/w3c/core-aam/issues/104 and assigned to @cookiecrook.

This one can be closed.

**Comment by aleventhal** [comment]
Actually, there is one remaining thing

**Comment by aleventhal** [comment]
FWIW, here's the Firefox issue to implement this: https://bugzilla.mozilla.org/show_bug.cgi?id=1726087

**Comment by jcsteh** [comment]
>     * "button-label" (when the name came from other markup such as aria-label)

What is this for? How can the description come from aria-label and why is it specific to buttons?

**Comment by JAWS-test** [comment]
> How can the description come from aria-label and why is it specific to buttons?

If the name of a button come from aria-label oder aria-labelledby, then the value of the button is the description: https://www.w3.org/TR/html-aam-1.0/#input-type-button-input-type-submit-and-input-type-reset-accessible-description-computation

**Comment by aleventhal** [comment]
That's correct. What @JAWS-test is saying is where Chrome is using kButtonLabel.

**Comment by jnurthen** [comment]
@spectranaut does this have tests?

**Comment by spectranaut** [comment]
I added a test: https://github.com/web-platform-tests/wpt/pull/34758

And here is a link to the webkit tracking issue (so they are all in one place): https://bugs.webkit.org/show_bug.cgi?id=206253#c18
